roboflow/core/version.py

```diff
@@ -438,13 +438,13 @@ def deploy(self, model_type: str, model_path: str, filename: str = "weights/best
             filename (str, optional): The name of the weights file. Defaults to "weights/best.pt".
         """
 
-        supported_models = ["yolov5", "yolov7-seg", "yolov8", "yolov9", "yolonas", "paligemma", "yolov10"]
+        supported_models = ["yolov5", "yolov7-seg", "yolov8", "yolov9", "yolonas", "paligemma", "yolov10", "florence-2"]
 
         if not any(supported_model in model_type for supported_model in supported_models):
             raise (ValueError(f"Model type {model_type} not supported. Supported models are" f" {supported_models}"))
 
-        if "paligemma" in model_type:
-            self.deploy_paligemma(model_type, model_path, filename)
+        if "paligemma" in model_type or "florence-2" in model_type:
```

Comment:
As I understand, model_types should start with `paligemma`/`florence-2`. Python `.startswith` accept a tuple of prefixes. Small improvement, but make the decision clear.

```suggestion
        if model_type.startswith(("paligemma", "florence-2")):
```